### ISSUE TITLE
Script to ingest data into local server

### ISSUE THREAD

**Issue by peterfpeterson** [mantidproject/webapp#7]
Information from http://reports.mantidproject.org/api/usage and http://reports.mantidproject.org/api/feature are needed to do better testing for development. Write a script using [python requests](http://docs.python-requests.org/en/master/) to copy information from the production site. Meta-code:
```python
req = requests.get('http://reports.mantidproject.org/api/usage')
doc = req.json()

for item in doc:
    requests.post(`http://127.0.0.1:8000/api/usage', item)
```
Something similar for http://reports.mantidproject.org/api/feature as well.

**Comment by peterfpeterson** [comment]
1. Guard the outermost level with
```python
if __name__ == "__main__":
```
2. Add a first line of
```shell
#!/usr/bin/env python
```
3. Instead of writing your own command line parser, use [argparse](https://docs.python.org/2/howto/argparse.html).
4. Pass `get_url` and `post_url` as arguments to main.
5. Add an option for number of pages to fetch.

**Comment by peterfpeterson** [comment]
This was fixed in #9.
